.rat-excludes

```diff
@@ -83,3 +83,5 @@ gen-java.*
 .*avpr
 org.apache.spark.sql.sources.DataSourceRegister
 .*parquet
+spark-deps-hadoop1
```

Comment:
I think this is a limitation in the CLI version of RAT: as near as I can tell, its exclude file regexes operate on files / directories rather than complete paths.